contributing/README.md

```diff
@@ -44,10 +44,11 @@ Developing without Vagrant
    ```sh
    $ [ "$(ulimit -n)" -lt 1024 ] && ulimit -n 1024
    ```
-1. Verify you can run tests
+1. Verify you can run smoke tests
    ```sh
    $ make test
    ```
+   **Note:** You can think of this as a `smoke` test which runs a subset of tests and some may fail because of `operation not permitted` error which requires `root` access. You can use `go test` to test the specific subsystem which you are working on and let the CI run rest of the tests for you.
```

Comment:
We're unfortunately not 100% consistent with it everywhere, but we try to hard-wrap long lines in markdown like this:

```suggestion
   **Note:** You can think of this as a `smoke` test which runs a subset of
   tests and some may fail because of `operation not permitted` error which
   requires `root` access. You can use `go test` to test the specific subsystem
   which you are working on and let the CI run rest of the tests for you.
```